cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,51 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import { faker } from '@faker-js/faker';
+import ArticlePageObject from '../support/pages/article.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
 
+const testData = {
+  title: faker.lorem.words(),
+  description: faker.lorem.words(),
+  text: faker.lorem.sentence(),
+  tag: faker.lorem.word()
+};
+
+describe('Article', () => {
   beforeEach(() => {
     cy.task('db:clear');
+
+    cy.registrationAndAuthorization();
+
+    articlePage.visit();
   });
 
   it('should be created using New Article form', () => {
+    articlePage.typeTitle(testData.title);
+    articlePage.typeDespription(testData.description);
```

Comment:
There seems to be a typo in the method name `typeDespription`. It should likely be `typeDescription`. Ensure that the method name matches the implementation in the `ArticlePageObject` class.